pkg/reader/node/node.go

```diff
@@ -11,42 +11,73 @@ import (
 	"github.com/cilium/tetragon/pkg/option"
 )
 
+const (
+	hubbleNodeNameEnvVar = "HUBBLE_NODE_NAME"
```

Comment:
How about `TETRAGON_NODE_NAME`?

Reply:
Good call!

Happy to add that as the preferred and keep the following order;
```yaml
export:
- TETRAGON_NODE_NAME
- HUBBLE_NODE_NAME
- NODE_NAME
- hostname
k8s:
- NODE_NAME
- TETRAGON_NODE_NAME
- HUBBLE_NODE_NAME
- hostname
```

And then we can potentially drop the old HUBBLE_NODE_NAME after a while, so we avoid any breaking changes with this PR. Does that sound sane?

Reply:
Let's leave it for a followup. Disregard my first comment.

Reply:
Sounds good

Reply:
> And then we can potentially drop the old HUBBLE_NODE_NAME after a while, so we avoid any breaking changes with this PR. Does that sound sane?

That sounds perfect, thanks!